examples/get-started/pure-js/carto_styles/README.md

```diff
@@ -0,0 +1,8 @@
+Pure js demo for CARTO layers
```

Comment:
Get-started examples are boiler plates for new developers. They are not meant to demo every existing feature. This is better suited in carto's own example catalog?